File: app/src/main/java/com/czbix/v2ex/util/IoUtils.java
package com.czbix.v2ex.util;

import com.czbix.v2ex.AppCtx;

import java.io.File;

public class IoUtils {
    private static File cacheDir;

    static {
        cacheDir = AppCtx.getInstance().getCacheDir();
    }

    public static File getWebCachePath() {
        return new File(cacheDir, "webCache");
    }
}

File: app/src/main/kotlin/com/czbix/v2ex/util/IoUtils.kt
package com.czbix.v2ex.util

import com.czbix.v2ex.AppCtx

import java.io.File

object IoUtils {
    private var cacheDir: File

    init {
        cacheDir = AppCtx.instance.cacheDir
    }

    @JvmStatic
    val webCachePath: File by lazy {
        File(cacheDir, "webCache")
    }
}
